doc/installation.rst

```diff
@@ -4,85 +4,94 @@ Installation
 Step 1: Install a `Miniconda <https://conda.io/miniconda.html>`_  (or `Anaconda <https://www.anaconda.com/downloads>`_) environment
 -----------------------------------------------------------------------------------------------------------------------------------
 
-Any Linux, Mac OS X, or Windows computer with a web browser (preferably Google Chrome) should be suitable. 16GB of RAM is required for some of the examples, but most will run fine in 4GB.
+Any Linux, Mac OS X, or Windows computer with a web browser (preferably
+Google Chrome) should be suitable. 16GB of RAM is required for some of
+the examples, but most will run fine in 4GB.
 
-If you don't already have conda on your machine, you can get it from `conda.io <https://conda.io/miniconda.html>`_, and then open a terminal window.
+If you don't already have conda on your machine, you can get it from
+`conda.io <https://conda.io/miniconda.html>`_, and then open a terminal
+window.
 
-If you do have conda already, it's a good idea to update it (running it twice to get the very latest) to ensure you have the latest version::
+If you do have conda already, it's a good idea to update it (running it
+twice to get the very latest) to ensure you have the latest version::
 
    > conda update conda
    > conda update conda
 
-You should then create a separate Conda environment to work in for this tutorial::
-
-   > conda create -n holoviz-tutorial python=3.7
-   > conda activate holoviz-tutorial
-
-Depending on how your conda is set up, you may need to use "source" instead of "conda" activate, or sometimes just "activate" on Windows.
-
-
-Step 2: Install ``holoviz`` and other required libraries
-------------------------------------------------------
+You should then make sure you have `anaconda-project` installed after
+activating your chosen conda environment:
```

Comment:
"activating your chosen envt" is vague, given that these are instructions for new users. Which envt do I need?, they might wonder.  How do you activate it?. We could put in a link to the conda docs, but here we should probably just say that the remaining commands should be executed in a conda environment.  We should also emphasize that none of these commands will _change_ their conda environment, i.e. clarify that the result will be a new conda envt just for the tutorial.

Reply:
> ... that none of these commands will change their conda environment

Adding `anaconda-project` may well change their conda environment, but not radically. All the steps afterwards are isolated though.

I agree with your suggestion though but is hard to phrase sensibly without potentially confusing people...